lib/broccoli/angular-broccoli-compass.js

```diff
@@ -0,0 +1,66 @@
+/* jshint node: true, esversion: 6 */
+'use strict';
+
+let sass;
+let compass;
+try {
+  try {
+    sass = require('sass');
+    compass = require('compass');
+  } catch (e) {
+    sass    = require(`${process.env.PROJECT_ROOT}/node_modules/node-sass`);
+    compass = require(`${process.env.PROJECT_ROOT}/node_modules/compass-importer`);
+  }
+} catch (e) { 
+  sass = null;
+  compass = null;
+}
+
+const Plugin  = require('broccoli-caching-writer');
+const fse     = require('fs-extra');
+const path    = require('path');
+const Funnel  = require('broccoli-funnel');
+
+class CompassPlugin extends Plugin {
+  constructor(inputNodes, options) {
+    super(inputNodes, {});
+
+    options = options || {};
+    Plugin.call(this, inputNodes, {
+      cacheInclude: [/(.*?).scss$/, /(.*?).sass$/]
```

Comment:
I still believe that `.scss` should be `\.scss` (similar for `.sass` here and elsewhere)... :smile: